rules/naming/src/Naming/PropertyNaming.php

```diff
@@ -243,6 +282,65 @@ private function removeInterfaceSuffixPrefix(string $className, string $shortNam
         return $shortName;
     }
 
+    /**
+     * @return ClassMethod[]
+     */
+    private function getPrefixedClassMethods(Property $property): array
+    {
+        $name = $this->nodeNameResolver->getName($property);
+        if ($name === null) {
+            return [];
+        }
+
+        $classLike = $property->getAttribute(AttributeKey::CLASS_NODE);
+        if ($classLike === null) {
+            return [];
+        }
+
+        $classMethods = $this->betterNodeFinder->findInstanceOf($classLike, ClassMethod::class);
+        return array_filter($classMethods, function (ClassMethod $classMethod): bool {
+            $classMethodName = $this->nodeNameResolver->getName($classMethod);
+            return Strings::match($classMethodName, '#^(is|are|was|were|has|have|had|can)[A-Z].+#') !== null;
```

Comment:
Could you extract the 2nd arguments as regex constant with https://regex101.com/ link? Just so we have it covered.

It's in `phpstan.neon`, but ignored now. I'll make sure it's not ignored :)

Reply:
Ready